satellite/payments/stripecoinpayments/service.go

```diff
@@ -45,3 +45,20 @@ func (service *Service) Setup(ctx context.Context, userID uuid.UUID, email strin
 	// TODO: delete customer from stripe, if db insertion fails
 	return service.customers.Insert(ctx, userID, email)
 }
+
+// Balance returns an integer amount in cents that represents the current balance of payment account.
+func (service *Service) Balance(ctx context.Context, userID uuid.UUID) (_ int64, err error) {
+	defer mon.Task()(&ctx, userID)(&err)
+
+	customerID, err := service.customers.GetCustomerID(ctx, userID)
+	if err != nil {
+		return
```

Comment:
Prefer to avoid naked returns, instead return the values explicitly. It helps to avoid problems with variable shadowing.

Reply:
ok, thanks for advice!